andela/activo-web
# #159159935 Fix crashing application

**What does this PR do?**
Fixes the crashing application on both staging and develop

**Description of Task to be completed?**
- remove duplicate code arising from rebase gone wrong
- add id to additional attributes to prevent component from unmounting unnecessarily
- remove call to method handler that generates current time

**How should this be manually tested?**
After cloning, run `yarn start:dev`. The application should start as expected and everything work normally.

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
[#159159935](https://www.pivotaltracker.com/story/show/159159935)

**Screenshots (if appropriate)**

<img width="1280" alt="screen shot 2018-07-19 at 5 24 27 am" src="https://user-images.githubusercontent.com/24798364/42921473-11084ab8-8b14-11e8-9186-99ac3866dbbe.png">

<img width="1280" alt="screen shot 2018-07-19 at 5 24 30 am" src="https://user-images.githubusercontent.com/24798364/42921478-1a4b3ee6-8b14-11e8-9f78-a565ffad7a96.png">


**Questions:**

xxx
